cmux.go

```diff
@@ -162,18 +169,18 @@ func (l muxListener) Accept() (c net.Conn, err error) {
 	}
 }
 
-type MuxConn struct {
+type muxConn struct {
```

Comment:
I'm using it in an personal project which isn't public yet. :-) 

Ah, then please add something simple for now:

// MuxConn wraps a net.Conn and provides transparent sniffing of connection data.